pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -100,6 +103,9 @@
 public class PinotClientRequest {
   private static final Logger LOGGER = LoggerFactory.getLogger(PinotClientRequest.class);
 
+  @Inject
+  PinotConfiguration _brokerConf;
```

Comment:
You don't have to inject the broker config here as it's already in the `BaseBrokerRequestHandler` class.

Reply:
There is no getter right now to get the config. I can add that. However is that a better option than injecting the config ?

Reply:
> However is that a better option than injecting the config ?

Probably not. It doesn't make sense to add that into the interface and we don't want to depend on the base class.